lua/telescope/pickers.lua

```diff
@@ -512,6 +522,10 @@ function Picker:find()
 
         status_updater { completed = false }
         self._on_lines(...)
+
+        vim.api.nvim_win_call(self.results_win, function()
+          vim.cmd "redraw"
+        end)
```

Comment:
This doesn't actually work for me, but this does:

```lua
self:move_selection(self.sorting_strategy == "ascending" and 1 or -1)
```

Reply:
Emmm, that's strange. I will roll this back.

Reply:
maybe we can keep both?

Reply:
Putting `redraw` but in `vim.schedule` seems to work.

Reply:
Damm, now it's not working.

Sorry about this...

Reply:
Emm, then keep both?

Reply:
Trying to reverse engineer this. So far this seems to work:

```lua
        self:reset_selection()
```